firefox_media_tests/playback/test_video_playback_base.py

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from marionette_driver import Wait
+from marionette_driver.errors import TimeoutException
+
+from firefox_media_tests.utils import verbose_until
+from media_test_harness.testcase import MediaTestCase
+from media_utils.video_puppeteer import (playback_done, playback_started,
+                                         VideoPuppeteer, VideoException)
+
+
+class TestVideoPlaybackBase(MediaTestCase):
```

Comment:
I think this class fits nicely in `media_test_harness/testcase.py`

For consistency, could call it `VideoElementTestCase`